crypto3/libs/blueprint/include/nil/blueprint/zkevm_bbf/opcodes/add_sub.hpp

```diff
@@ -47,35 +47,58 @@ namespace nil {
                 using generic_component<FieldType, stage>::lookup_table;
             public:
                 using typename generic_component<FieldType,stage>::TYPE;
+                using integral_type = boost::multiprecision::number<boost::multiprecision::backends::cpp_int_modular_backend<257>>;
```

Comment:
This type is not from FieldType. It should be taken from zkevm_word.
We need it because zkevm_word is modular type and some bitwize and comparing operations are not available.

I defined zkevm_word_integral_type. 
It would be better